src/state/opamSwitchState.ml

```diff
@@ -101,15 +101,15 @@ let load lock_kind gt rt switch =
   let switch_config = load_switch_config gt switch in
   if OpamVersion.compare
       (OpamVersion.nopatch (switch_config.OpamFile.Switch_config.opam_version))
-      (OpamVersion.nopatch OpamFormatUpgrade.latest_version)
-     <> 0 then
+      (OpamVersion.nopatch OpamFormatUpgrade.v2_0)
+     < 0 then
```

Comment:
Maybe the value should be exported as `OpamFormatUpgrade.latest_forward_compatible_version` or something more generic like that ?

Reply:
Updated with `latest_compatible_switch_version`